src/Analyzer.JsonRuleEngine.UnitTests/ConfigurationsTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Templates.Analyzer.RuleEngines.JsonEngine.UnitTests
+{
+    [TestClass]
+    public class ConfigurationsTests
+    {
+        // Not sure what to put
```

Comment:
Maybe you can mock the configuration objects to run the JSON engine with that and validate the config features? Some end to end with test config files would be ideal too